### PR TITLE
slsa.dev v2.0 - Major overhaul

### PR DESCRIPTION
**Pull request by devmoran**
This PR contains a new layout for the slsa.dev website. The changes are wide-ranging and substantial, but have been seen and signed off by @kimsterv and @MarkLodato.

- Adds custom page template and styling
- Adds new documentation structure
- Creates updated project overview

|   Page|   Screenshot|
|---|---|
|  Overview | image  |
| Security levels  |  image |
|  Requirements |  image |
| Example of use |image |
| Get involved |image |

### CODE REVIEW

**Review by MarkLodato — CHANGES_REQUESTED** [review]
Thanks, @devmoran! The PR is difficult to review, so I think it makes sense to split into smaller ones.

I just sent out #114 to do the file renames. The directory must be named `docs/` since that is the only name supported by GitHub Pages. I also moved the CNAME and _config.yaml files into there as well, which again is required.

Could you base your changes off that? You might want to also squash all your changes into a single commit, but it's up to you.

Splitting it further, e.g. style changes vs content changes, is up to you. We can see how big the diff is.

**Comment by devmoran** [comment]
No problem @MarkLodato, I'll get this split out once #114 is merged.

**Review by MarkLodato — CHANGES_REQUESTED** [review]
I think this looks really great! Thanks so much for your work!

I'll be OOO until Monday, so I wanted to leave some early feedback now. Feel free to merge if others approve.

File: README.md [hunk 1]
```diff
@@ -1,334 +1,40 @@
...
-reasonable security guarantees. There is an urgent need for a solution in the
-face of the eye-opening, multi-billion dollar attacks in recent months (e.g.
-[SolarWinds](https://www.solarwinds.com/sa-overview/securityadvisory),
-[CodeCov](https://about.codecov.io/security-update/)), some of which could have
-been prevented or made more difficult had such a framework been adopted by
-software developers and consumers.
+## Project status
 
-SLSA addresses three issues:
+SLSA is currently in alpha. The framework is ready-to-use. It’s in use by Google and is constantly being improved.
```
[the remark this answers is not in the document]
Comment (inferno-chromium) [reply README.md:25]: yes should remove both and instead maybe something like "We encourage the community to try SLSA levels incrementally and share experience or problems they encounter" (feel free to improve wording).

File: README.md [hunk 2]
```diff
@@ -1,334 +1,40 @@
...
+- [Levels](SLSA/levels.md) (Defining the framework)
+- [Requirements](SLSA/requirements.md) (How to attain compliance)
+- [Example of use](SLSA/example.md)
+- Our [roadmap](SLSA/roadmap.md)
 
-**IMPORTANT:** SLSA is an evolving specification and we are looking for
-wide-ranging feedback via [GitHub issues], [email][mailing list], or
-[feedback form]. SLSA is being developed as part of the
-[OpenSSF Digital Identity WG](https://github.com/ossf/wg-digital-identity-attestation).
+## Maintainers
```
[the remark this answers is not in the document]
Comment (inferno-chromium) [reply README.md:16]: I would prefer to not have this section, we should just keep the openssf wg and calendar invite info. We can add a contributors section if we want to show the list of people that made this happen. But maintainers sections makes it feel less included, so we want to encourage community contributions.

File: SLSA/index.md [hunk 3]
```diff
@@ -0,0 +1,146 @@
...
+
+**NOTE: This site is best viewed at https://slsa.dev.**
+
+<!--{% endif %}-->
+
+<span class="subtitle">
+
+SLSA [sal•sa] is **Supply-chain Levels for Software Artifacts.**
+
+An open source security framework to describe and verify what integrity looks like, giving anyone working with software a common language and a way to work at scale.
```
[the remark this answers is not in the document]
Comment (inferno-chromium) [reply SLSA/index.md:13]: This new definition has been slightly confusing to me, i really liked the old version we had ?

**Comment by devmoran** [comment]
Thanks @MarkLodato and @inferno-chromium for the notes and feedback. I'll address them all individually (there's a couple I want to discuss with Steven next week once he's back from vacay), but otherwise I'll incorporate these changes into the new separated PRs.

**Pull request closed without merging**